projects/packages/search/src/dashboard/store/actions/jetpack-settings.js

```diff
@@ -27,15 +27,15 @@ export const TOGGLE_SEARCH_MODULE = 'TOGGLE_SEARCH_MODULE';
 export function* updateJetpackSettings( settings, oldSettings ) {
 	try {
 		yield updatingNotice();
-		yield setJetpackSettings( settings );
 		yield setUpdatingJetpackSettings();
+		yield setJetpackSettings( settings );
 		yield updateJetpackSettingsControl( settings );
 		const updatedSettings = yield fetchJetpackSettings();
 		yield setJetpackSettings( updatedSettings );
-		return successNotice( __( 'Updated settings.' ) );
+		return successNotice( __( 'Updated settings.', 'jetpack' ) );
 	} catch ( e ) {
 		yield setJetpackSettings( oldSettings );
-		return errorNotice( __( 'Error Update settings…' ) );
+		return errorNotice( __( 'Error Update settings…', 'jetpack' ) );
```

Comment:
```suggestion
		return errorNotice( __( 'Error updating settings…', 'jetpack' ) );
```

Reply:
Noticed this message when ngrok 403-ed on the settings endpoint.

Reply:
thanks for catching this, addressed in 76bc4f5703